x-pack/test/observability_ai_assistant_api_integration/tests/conversations/conversations.spec.ts

```diff
@@ -250,5 +250,113 @@ export default function ApiTest({ getService }: FtrProviderContext) {
+              params: {
+                body: {
+                  conversation: conversationCreate,
+                },
+              },
+            })
+            .expect(200);
+        });
+
+        after(async () => {
+          await observabilityAIAssistantAPIClient
+            .editor({
+              endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
+              params: {
+                path: {
+                  conversationId: createResponse.body.conversation.id,
+                },
+              },
+            })
+            .expect(200);
+        });
+
+        it('POST /internal/observability_ai_assistant/conversation', async () => {
+          try {
+            await observabilityAIAssistantAPIClient.unauthorizedUser({
+              endpoint: 'POST /internal/observability_ai_assistant/conversation',
+              params: {
+                body: {
+                  conversation: conversationCreate,
+                },
+              },
+            });
+          } catch (e) {
+            expect(e.status).to.be(403);
+          }
+        });
+
+        it('POST /internal/observability_ai_assistant/conversations', async () => {
+          try {
+            await observabilityAIAssistantAPIClient.unauthorizedUser({
+              endpoint: 'POST /internal/observability_ai_assistant/conversations',
+            });
+          } catch (e) {
+            expect(e.status).to.be(403);
+          }
+        });
+
+        it('PUT /internal/observability_ai_assistant/conversation/{conversationId}', async () => {
+          try {
+            await observabilityAIAssistantAPIClient.unauthorizedUser({
+              endpoint: 'PUT /internal/observability_ai_assistant/conversation/{conversationId}',
+              params: {
+                path: {
+                  conversationId: createResponse.body.conversation.id,
+                },
+                body: {
+                  conversation: merge(omit(conversationUpdate, 'conversation.id'), {
+                    conversation: { id: createResponse.body.conversation.id },
+                  }),
+                },
+              },
+            });
+          } catch (e) {
+            expect(e.status).to.be(403);
+          }
+        });
+
+        it('GET /internal/observability_ai_assistant/conversation/{conversationId}', async () => {
+          try {
+            await observabilityAIAssistantAPIClient.unauthorizedUser({
+              endpoint: 'GET /internal/observability_ai_assistant/conversation/{conversationId}',
+              params: {
+                path: {
+                  conversationId: createResponse.body.conversation.id,
+                },
+              },
+            });
+          } catch (e) {
+            expect(e.status).to.be(403);
+          }
+        });
+
+        it('DELETE /internal/observability_ai_assistant/conversation/{conversationId}', async () => {
+          try {
```

Comment:
You may want to have `expect.assertions`. This ensures that the test actually asserts something. For instance, if by mistake the `unauthorizedUser` indeed does have access to the route, the test will still pass as it is now, because we'll never end up in the catch block. 

```suggestion
          expect.assertions(1);
          try {
```

Reply:
... same with the other ones. There are other ways to prevent this problem. For instance throwing an explicit error:

```ts
          try {
            await observabilityAIAssistantAPIClient.unauthorizedUser({
              endpoint: 'DELETE /internal/observability_ai_assistant/conversation/{conversationId}',
              params: {
                path: {
                  conversationId: createResponse.body.conversation.id,
                },
              },
            });
            throw new Error('Expected to throw');
          } catch (e) {
            expect(e.status).to.be(403);
          }
```

Reply:
`expect.assertions(1);` is not available via `'@kbn/expect`.

<img width="923" alt="image" src="https://github.com/user-attachments/assets/bce76207-bcf0-4fd4-bef6-2915051ddbd6">

I'lll try the alternative

Reply:
Done